crates/swc_ecma_loader/src/resolvers/node.rs

```diff
@@ -294,14 +309,17 @@ impl NodeModulesResolver {
         base_dir: &Path,
         target: &str,
     ) -> Result<Option<PathBuf>, Error> {
-        let mut path = Some(base_dir);
+        let absolute_path = to_absolute_path(base_dir).unwrap();
+        let mut path = Some(Path::new(absolute_path.to_str().unwrap()));
```

Comment:
This is supper inefficient and wrong.
Please use `Some(&*absolute_path)` instead

Reply:
This point directly to the `inner` in the struct ?

Reply:
4bcc044b013a76a1974e93506d7b1ece577cfd5a should fix the issue